src/components/TeaserFront/Tile.js

```diff
@@ -162,32 +177,40 @@ const Tile = ({ children, attributes, image, alt, onClick, color, bgColor, align
     IMAGE_SIZE.large,
     false
   )
+  let containerStyle = {
+    background,
+    cursor: onClick ? 'pointer' : 'default',
+    justifyContent
+  }
+  if (imageOnly) {
+    containerStyle = {
+      ...containerStyle,
+      padding: 0
+    }
+  }
+  const imageStyle = imageOnly ? {maxHeight: '100%', maxWidth: '100%'} : {}
```

Comment:
mv to styles and use the same pattern as with `imageOnlyContainer` vs. `imageContainer`